packages/nextjs/package.json

```diff
@@ -81,7 +81,7 @@
     "@types/node": "^18.19.33",
     "@types/react": "*",
     "@types/react-dom": "*",
-    "next": "^13.5.4",
+    "next": "^14.1.1",
```

Comment:
⛔ We should drop this change since we want to run the tests with our minimum supported `next` version which is `13.5`